src/main/java/org/assertj/core/api/AbstractLongArrayAssert.java

```diff
@@ -198,6 +202,34 @@ public SELF contains(long... values) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual array contains the values of the given array, in any order.
+   * <p>
+   * Example:
+   * <pre><code class='java'> // assertions will pass
+   * assertThat(new long[] { 1L, 2L, 3L }).contains(new Long[] { 1L, 2L });
+   * assertThat(new long[] { 1L, 2L, 3L }).contains(new Long[] { 3L, 1L });
+   * assertThat(new long[] { 1L, 2L, 3L }).contains(new Long[] { 1L, 3L, 2L });
+   *
+   * // assertions will fail
+   * assertThat(new long[] { 1L, 2L, 3L }).contains(new Long[] { 1L, 4L });
+   * assertThat(new long[] { 1L, 2L, 3L }).contains(new Long[] { 4L, 7L });</code></pre>
+   *
+   * @param values the given {@code Long} array of values.
+   * @return {@code this} assertion object.
+   * @throws NullPointerException if the given argument is {@code null}.
+   * @throws IllegalArgumentException if the given argument is an empty array.
+   * @throws AssertionError if the actual array is {@code null}.
+   * @throws AssertionError if the actual array does not contain the given values.
+   * @since 3.19.0
+   */
+  public SELF contains(Long[] values) {
+    requireNonNull(values, shouldNotBeNull("values").create());
```

Comment:
maybe extract a method `requireNonNull(Long[] values, String parameterDescription)` that would call `requireNonNull(values, shouldNotBeNull(parameterDescription).create())`?